src/Libraries/Tesellation/Tessellation.csproj

```diff
@@ -15,8 +15,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.18.0.1400" />
-    <PackageReference Include="MIConvexHull" version="1.1.17.0411" CopyPDB="true" />
-	<PackageReference Include="StarMath" version="2.0.14.1114" CopyPDB="true" />
+    <PackageReference Include="MIConvexHull" version="1.1.17.411" CopyPDB="true" />
+	<PackageReference Include="StarMath" version="2.0.20.204" CopyPDB="true" />
```

Comment:
latest, supports netstandard 2.1

Reply:
net std 2.1 does not support net.48 - see this:
<img width="862" alt="Screenshot 2023-06-29 at 9 56 05 AM" src="https://github.com/DynamoDS/Dynamo/assets/508936/52e003aa-7478-457a-8e10-56a3b40cad58">

only net std 1 - 2 support net framework.

Reply:
Ok, thanks for pointing that out.